app/models/post.rb

```diff
@@ -0,0 +1,14 @@
+class Post < ApplicationRecord
+  has_one :users
+  has_many :comments
+  has_many :likes
+
+  def last_comments(the_post_id)
+    Comment.where("posts_id = #{the_post_id}", params[:posts_id]).limit(5).order(created_at: :desc)
+  end
+
+  def update_count(u_name, value)
+    p_count = User.find_by(name: u_name.to_s)
+    p_count.update(PostsCounter: value)
+  end
```

Comment:
**Optional Changes**
- [ ] Good effort with this function. You could consider using  [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - **after_save** to call the #update_post_counter method every time a new post has been saved to database, then make the update_post_counter method private